teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name, parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] in the initialize method apart from `classroom` you also need to pass `age`, `name` and `parent_permission` as arguments to make it work.